lib/perl/Genome/Model/Build/Command/Export.pm

```diff
@@ -97,6 +115,14 @@ sub execute {
 
     $find->($export_directory);
 
+    if($self->create_tarball){
+        my $tarball = File::Spec->join($self->target_export_directory, "build" . $self->build->id . ".tar");   
+        my $rv = Genome::Sys->shellcmd(cmd => ["tar","-cf",$tarball,"-C",$tempdir,"build" . $self->build->id]);
```

Comment:
Could also add a this key to `shellcmd`, and it'll double-check the file was created with size:
```perl
   output_files => [ $tarball ],
````

Reply:
did this as well